src/app/security/maturity-model/components/CoreFunctions.tsx

```diff
@@ -0,0 +1,17 @@
+import { coreFunctionsData } from '../data/coreFunctionsData'
+
+export function CoreFunctions() {
+  return (
+    <div className="prose first:prose-h3:mt-0 prose-h4:text-brand-300 prose-h5:text-brand-300">
+      {coreFunctionsData.map(({ title, Content }, index) => (
+        <article key={index}>
```

Comment:
```suggestion
        <article key={title}>
```
https://react.dev/learn/rendering-lists#why-does-react-need-keys - they recommend not using index as a key